lib/webrtc/call.js

```diff
@@ -145,6 +151,64 @@ MatrixCall.prototype.placeScreenSharingCall =
 };
 
 /**
+ * Play the given HTMLMediaElement, serialising the operation into a chain
+ * of promises to avoid racing access to the element
+ * @param {Element} HTMLMediaElement to play
+ * @param {string} Arbitrary ID to track the chain of promises to be used
```

Comment:
The format for JSDoc is:
```
@param {Type} argName Some description
```
The `argName` is missing here. This is important because `npm run gendoc` makes HTML output based on the JSDoc here which is shipped to gh-pages on releases.